site/docs/advanced/README.md

```diff
@@ -1,5 +1,5 @@
 ---
-next: ./middleware.md
+next: ./middleware.html
```

Comment:
Do they do it that way? It looks to me like the extensions are transpiled properly here, too.

Reply:
![image](https://user-images.githubusercontent.com/49933115/168445747-d17d908a-ddff-4689-91e1-9d1fb4d44bd5.png)

Reply:
I saw this.

Reply:
I saw 
![image](https://user-images.githubusercontent.com/12952387/168445852-a33c7bf5-7499-4ab8-83cf-22b3d4dd3ce3.png)
on https://grammy.dev/guide/getting-started.html#getting-started-on-deno which [uses](https://github.com/grammyjs/website/blob/5ea6449081558b4b70933eeced684cac7b9f91bf/site/docs/guide/getting-started.md?plain=1#L2) `.md`.

Reply:
Alrighty.

Reply:
I think it does not really matter. They use HTML, but MD will be transpiled anyway, so the outcome really is identical. Pick whatever you prefer

Reply:
I was wrong about the documentation. I picked what they prefer.